syntax/sh.nanorc

```diff
@@ -8,7 +8,8 @@ color green "(\{|\}|\(|\)|\;|\]|\[|`|\\|\$|<|>|!|=|&|\|)"
 color green "-[Ldefgruwx]\>"
 color green "-(eq|ne|gt|lt|ge|le|s|n|z)\>"
 color brightblue "\<(cat|cd|chmod|chown|cp|echo|env|export|grep|install|let|ln|make|mkdir|mv|rm|sed|set|tar|touch|umask|unset)\>"
-icolor brightgreen "^\s+?[0-9A-Z_]+\s+?\(\)"
+#icolor brightgreen "^\s+?[0-9A-Z_]+\s+?\(\)"
+icolor brightgreen "^\s+[0-9A-Z_]+\s+\(\)"
```

Comment:
Is this dependent on the version of nano? `\s+?` is any amount of space, lazy. I personally run the latest (svn) and this is not an issue for me. I can confirm that v2.2.6 crashes on OS-X, but since this is a person fork with no intention of merging up-stream I won't accept this hunk regardless. I don't use this repository on OS-X.